addons/point_of_sale/static/src/js/Popups/CashOpeningPopup.js

```diff
@@ -37,14 +36,14 @@ odoo.define('point_of_sale.CashOpeningPopup', function(require) {
         updateCashOpening({ total, moneyDetailsNotes }) {
             this.state.openingCash = total;
             if (moneyDetailsNotes) {
-                this.state.notes = moneyDetailsNotes;
+                if (this.state.notes) this.state.notes += '\n';
+                this.state.notes += _.str.sprintf(this.env._t('Opening %s'), moneyDetailsNotes);
```

Comment:
This means that whenever we re-update the cash detail, the notes are increasing. Is that the behavior that we are looking for ? I know that by default we wanted to reset everything whenever it was re-updated

Reply:
I'd suggest you ask the PO about that and you confront him with those two scenarios

Reply:
Indeed, I asked @dama-odoo his opinion on this one!

Reply:
Hi @trj-odoo, I have dropped the related commit and reverted to the previous behavior, as suggested by the PO. Thanks!